source/about/cloud-subscriptions.rst

```diff
@@ -0,0 +1,114 @@
+
+Renew your subscription
+-----------------------
+
+Monthly Cloud subscriptions renew automatically.
+
+Frequently Asked Questions
+---------------------------
+
+How am I billed for my Cloud monthly subscription?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Your first Cloud subscription charge is at the end of the calendar month, and all monthly charges are billed in arrears on the first day of each month, e.g., June billing will be done on 7/1 for the period 6/1 - 6/30. 
+
+After your first month, the billing period begins at 12 AM UTC on the first day of each calendar month and ends at 11:59 PM UTC on the final day of the same calendar month. Per-user pricing for each billing period will be based on the number of registered users.
+
+If you begin your subscription in the middle of a billing period, charges will be prorated. For example, if you signed up 6/15, we'll charge you on 7/1 for the period 6/15 - 6/30.
+
+Similarly, if you end your subscription in the middle of a billing period, charges will be prorated. For example, if you cancel your account on 6/15, we'll charge you on 7/1 for the period 6/1 - 6/15.
+
+If you upgrade in the middle of a billing period, charges will also be prorated. For example, if you upgrade from Starter to Professional on 6/15, we'll charge you 7/1. This will be a single charge for Starter for the period 6/1 - 6/14, then a separate charge for Professional for the period 6/15 - 6/30.
+
+Can I purchase an annual subscription to Mattermost Cloud deployments?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes. Annual Cloud subscriptions need to be processed by our Sales team, as we don't currently support an annual term using the self-service Customer Portal or Cloud workspace.
+
+If you upgrade during a monthly billing cycle, charges will be prorated. For example, if you upgrade from monthly Starter to an annual term of Mattermost Professional on 6/15, we'll charge you on 7/1 for Starter for the period 6/1 - 6/14. A separate invoice will be provided for the annual Mattermost Professional subscription period starting 6/15.
+
+How do I cancel my subscription? 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To cancel your subscription, please `contact us <https://customers.mattermost.com/cloud/contact-us>`__.
+
+How much does Mattermost Cloud cost?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Mattermost Cloud Professional is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users, please contact us for a quote.
+* Mattermost Cloud Enterprise is an annual subscription. Please contact us for a quote.
+
+How can customers trial Mattermost Cloud?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Mattermost Cloud Professional plan includes a 14-day free trial.
```

Comment:
Is this section up-to-date?

Reply:
I removed this as it felt a bit redundant to the opening line of cloud begins with a 14 day trial.